app/assets/stylesheets/partials/_header.css.scss

```diff
@@ -0,0 +1,20 @@
+header.navbar {
+  background-color: #232120;
+  border-radius: 0;
+
+  ul.navbar-nav li a, .nav-header a {
+    color: #f8f8f8;
+  }
+
+  ul.navbar-nav li a:hover {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4